src/monexamine.cpp

```diff
@@ -645,13 +645,9 @@ bool monexamine::pet_menu( monster &z )
     std::string pet_name = z.get_name();
 
     amenu.text = string_format( _( "What to do with your %s?" ), pet_name );
-    if( z.has_flag( mon_flag_EATS ) && ( z.amount_eaten < ( z.stomach_size / 10 ) ) ) {
-        amenu.text = string_format( _( "What to do with your %s?\n" "Hunger: Famished" ), pet_name );
-    } else if( z.has_flag( mon_flag_EATS ) && ( z.amount_eaten > ( z.stomach_size / 10 ) &&
-               z.amount_eaten < z.stomach_size ) ) {
-        amenu.text = string_format( _( "What to do with your %s?\n" "Hunger: Hungry" ), pet_name );
-    } else if( z.has_flag( mon_flag_EATS ) && z.amount_eaten >= z.stomach_size ) {
-        amenu.text = string_format( _( "What to do with your %s?\n" "Hunger: Full" ), pet_name );
+    if( z.has_flag( mon_flag_EATS ) ) {
+        amenu.text = string_format( _( "What to do with your %s?\n" "Fullness: %i%%" ), pet_name,
```

Comment:
Note: This is a UI-functional change

It could alternatively be done by a call to a switch function that handles it